go/teams/seitan.go

```diff
@@ -39,17 +39,17 @@ const base30BitMask = byte(0x1f)
 type SeitanIKey string
 
 // "Packed Encrypted Invite Key"
-// All following 3 structs should be considerd one. When any changes,
+// All following 3 structs should be considered one. When any changes,
 // Version in PEIKey has to be bumped up.
 type SeitanPEIKey struct {
-	_struct               bool `codec:",toarray"`
-	Version               uint
-	TeamKeyGeneration     keybase1.PerTeamKeyGeneration
-	RandomNonce           keybase1.BoxNonce
-	EncryptedIKeyAndLabel []byte // keybase1.SeitanIKeyAndLabel MsgPacked and encrypted
+	_struct              bool `codec:",toarray"`
+	Version              uint
+	TeamKeyGeneration    keybase1.PerTeamKeyGeneration
+	RandomNonce          keybase1.BoxNonce
+	EncryptedKeyAndLabel []byte // keybase1.SeitanKeyAndLabel MsgPacked and encrypted
 }
 
-func GenerateIKey() (ikey SeitanIKey, err error) {
+func generateIKey(plusOffset int) (buf []byte, err error) {
```

Comment:
Since both consumers cast the result to `string`, maybe lets return string from this function.

I'm not a fan of having two functions named same but with lowercase, but thats debatable

Reply:
done! do you have any suggestions for the helper function name?